openmdao/docs/features/building_blocks/components/add_subtract_comp.rst

```diff
@@ -11,6 +11,12 @@ AddSubtractComp
 
     result = a * \textrm{scaling factor}_a + b * \textrm{scaling factor}_b + c * \textrm{scaling factor}_c + ...
 
+Method Signature
+----------------
+
+.. automethod:: openmdao.components.add_subtract_comp.AddSubtractComp.add_equation
+    :noindex:
+
```

Comment:
I think this signature should come _after_ the following section which introduces the method...